src/Workspaces/CSharp/Portable/Formatting/CSharpFormattingOptions.cs

```diff
@@ -146,6 +146,13 @@ public static partial class CSharpFormattingOptions
                 EditorConfigStorageLocation.ForBoolOption("csharp_indent_case_contents"),
                 new RoamingProfileStorageLocation("TextEditor.CSharp.Specific.IndentSwitchCaseSection")});
 
+        public static Option<bool> IndentSwitchCaseSectionWhenBlock { get; } = new Option<bool>(
+            nameof(CSharpFormattingOptions),
+            nameof(IndentSwitchCaseSectionWhenBlock), defaultValue: true,
+            storageLocations: new OptionStorageLocation[] {
+                EditorConfigStorageLocation.ForBoolOption("csharp_indent_case_contents_when_block"),
```

Comment:
Do we also need to update the AutomationObject for this one, for .vssettings file support?